Simple example advertisers

mharper opened the pull request.

mharper committed:
WIP: Simple example Eddystone URL advertiser

mharper commented:
@syoder am I on the right track?

syoder commented on examples/advertiser.go:
```diff
@@ -0,0 +1,19 @@
+	for {
+		adv.AdvertiseMfgData(0xfeaa, advert)
+		time.Sleep(5 * time.Second)
+	}
```
I don't think you need to call `AdvertiseMfgData` in a loop.  I think once you set it, it will advertise on whatever interval the hardware is set for.

Probably what you need to do after calling `AdvertiseMfgData` is wait on an os signal (sigterm or sigint) and then when received call `StopAdvertising` and then exit.  I think this is how you'd wait on sigterm or sigint:
```go
sigChan := make(chan os.Signal, 1)
signal.Notify(sigChan, syscall.SIGINT)
signal.Notify(sigChan, syscall.SIGTERM)
<-sigChan // wait for signal
```

syoder replied:
Oh, Eddystone-URL is a service advertisement, so you should be using `AdvertiseServiceData` instead of `AdvertiseMfgData`.

syoder commented:
Yes, you're on the right track.  Just a few minor changes.

mharper committed:
Fixes advertising example for Eddystone URL.

mharper commented:
OK, I can now see this Eddystone URL advertisement using the Beacon Tools app on iOS!

syoder commented:
It shows in Locate but not in the Chrome extension...

[several minutes pass]

ah, the url `https://radiusnetworks.com` doesn't work (try it in your browser if you don't believe me).  You have to change it to `https://www.radiusnetworks.com`.  Then it shows in the Chrome extension.

mharper committed:
Prepends `www` to the example advertiser URL

Turns out that `https://radiusnetworks.com` doesn't actually point
anywhere.  Consequently, the advertised URL wouldn't appear in the
Chrome widget on iOS.  Adding `www` makes it all work.

mharper commented:
![img_0543](https://cloud.githubusercontent.com/assets/85054/22163527/0abcf802-df08-11e6-974c-e08b32993f9e.PNG)
